storage/google/cloud/storage/blob.py

```diff
@@ -432,8 +432,10 @@ def download_to_filename(self, filename, client=None):
         with open(filename, 'wb') as file_obj:
             self.download_to_file(file_obj, client=client)
 
-        mtime = time.mktime(self.updated.timetuple())
-        os.utime(file_obj.name, (mtime, mtime))
+        updated = self.updated
```

Comment:
Because `self.updated` is a `@property` does a dictionary lookup and a string->native datetime conversion on every call. So calling it twice would be "double computation". (I figured someone would ask.)